package.json

```diff
@@ -56,7 +56,16 @@
     },
     "directories": {
       "buildResources": "resources"
-    }
+    },
+    "protocols": [
+      {
+        "name": "jitsi-protocol",
+        "role": "Viewer",
+        "schemes": [
+          "jitsi"
```

Comment:
can we please make this jitsi-meet ?

Reply:
I can see you already found the answer :) 
Sure thing, just pushed the change commit.